sdk/typescript/package.json

```diff
@@ -29,7 +29,8 @@
     "test": "pnpm test:unit && pnpm test:e2e",
     "test:unit": "vitest unit",
     "test:e2e": "wait-on http://127.0.0.1:9123 -l --timeout 120000 && vitest e2e",
-    "prepare:e2e": "cargo build --bin sui-test-validator --bin sui --profile dev && RUST_LOG=info,sui=debug,anemo_tower=warn cargo run --bin sui-test-validator",
+    "test:e2enw": "vitest e2e",
```

Comment:
It's not clear to me what `e2enw` means.

Reply:
`nw` means "no wait". I'll make it more explicit